web/src/layout/leftpanel/countrypanel.jsx

```diff
@@ -193,7 +194,7 @@ const CountryHeader = ({ parentPage, zoneId, data, isMobile, isDataAggregated })
 const CountryPanel = ({ electricityMixMode, isMobile, tableDisplayEmissions, zones }) => {
   const [tooltip, setTooltip] = useState(null);
   const { __ } = useTranslation();
-
+  const { failedRequestType } = useSelector((state) => state.data);
```

Comment:
For performance, as far as I understand you'd want to avoid destructuring in the redux selector and instead go directly to the object you need

https://stackoverflow.com/questions/59072200/useselector-destructuring-vs-multiple-calls